spec/models/organization_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Organization, type: :model do
+  let(:organization) { Organization.new(name: 'Name', creator_id: 1, monthly_point: 100, logo: 'logo.jpg')}
+
+  it 'is valid with correct data' do
+    expect(organization.valid?).to be_truthy
+  end
+
+  it 'is invalid with incorrect data' do
+    # validates in model should reject nil in fields creator_id, monthly_point
+    organization.creator_id = nil
+    expect(organization.invalid?).to be_truthy
+    organization.creator_id = 1
+    organization.monthly_point = nil
```

Comment:
don't write so many rows, create invalid_attributes for the organization and merge it with your variable for getting organization with invalid params  or create a separate variable in a separate context